blockchain/genesis.go

```diff
@@ -0,0 +1,46 @@
+package blockchain
+
+import (
+	"time"
+)
+
+// Genesis creates the Genesis block and returns is.
+//
+// Properties of the Genesis block:
```

Comment:
Take a look at the genesis block properties. These are pretty standard across cryptocurrencies. Let me know if you think anything needs to be added.